app/assets/javascripts/components/revisions/revision.jsx

```diff
@@ -3,37 +3,64 @@ import PropTypes from 'prop-types';
 import moment from 'moment';
 import DiffViewer from './diff_viewer.jsx';
 import CourseUtils from '../../utils/course_utils.js';
+import createReactClass from 'create-react-class';
 
-const Revision = ({ revision, wikidataLabel, course }) => {
```

Comment:
It's not a big deal either way, but is it necessary to convert this from a function component to a class component?